composer.json

```diff
@@ -99,7 +99,10 @@
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget",
+            "npm install",
+            "bower install",
+            "grunt"
```

Comment:
This could still be a good idea. Look at how I've managed to check for CS & for translated file in one extra build. I prefer one more build instead of all builds taking more than 7 minutes to run ..